pycds/climate_baseline_helpers.py

```diff
@@ -112,11 +112,14 @@ def load_pcic_climate_baseline_values(session, var_name, source):
 
         var_name (str): name of climate baseline variable for which the values are to be loaded
 
-        source (iterable): an interable that returns a sequence of fixed-width formatted ASCII lines
+        lines (iterable): an interable that returns a sequence of fixed-width formatted ASCII lines
             (strings) containing the data to be loaded; typically result of `file.readlines()`
 
+        network_name (str): name of the network to which the climate variable (identified by var_name)
+            must be associated
+
     Returns:
-        None
+        (n_added, n_skipped): counts of lines (not values!) added to database or skipped, respectively
```

Comment:
What is the difference between lines and values? Are there multiple values per line in the input file?